### DISCUSSION TITLE
Copilot approved

### DISCUSSION THREAD

**Discussion by nickkvolpe** [community/community#138592] [GitHub Education]
### Hi everyone,

I haven't gotten my Copilot yet and I applied and got accepted Monday night. How should I proceed?

**Comment by davevad93** [comment]
Hi @nickkvolpe ,

Don't worry, you don't have to do anything, just wait. When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @nickkvolpe was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by pg56714** [comment]
Mark me as answer thanks

@nickkvolpe 
[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)
